src/font.ts

```diff
@@ -31,6 +31,12 @@ export interface FontGlyph {
 }
 
 class Font {
+  /** Default sans-serif font family. */
+  static SANS_SERIF: string = 'Arial, sans-serif';
```

Comment:
I added these two constants. Developers who use VexFlow can set them to something different, and it will be reflected throughout their score whenever text is shown.

Reply:
No change required.